ocviapy/__init__.py

```diff
@@ -543,6 +543,16 @@ def details_str(self):
             )
         return detail_msg
 
+    @property
+    def image_pull_error(self):
```

Comment:
Should we check `if self.restype == "pod"` here and always return `False` if this is not a pod?